INTEGRATION_GUIDE.md

```diff
@@ -101,6 +101,36 @@ method. It can be called multiple time throughout the lifecycle of the SDK - eve
 
 While in the SDK's UI, users can exit prematurely by tapping on the "Done" button. They might do so too soon and therefore see that they need to go back and upload some more pictures. Unfortunately, this will happen after the `switchSdkDidComplete` delegate method call. So if you want to allow users to return to the SDK (And you should), make sure to hold on to the Switch SDK UI (the result from the `sdk.instantiateSwitchViewController()` statement) until you are sure users will not want to go back. After that, just clear the reference to the view controller and let ARC do the rest.
 
+### Sending Feedback
+
+It is always possible that the extractions our services find in the document are not correct. We are continuously working on improving our algorithms and to achieve that we need you!
+
+If you find an error in the extractions at a later stage in your app, you should send us "Feedback" - the correct values. This will help us train our extractors to better read documents.
+
+In order to send feedback, there are several things to consider. After you get the extraction collection from your `GiniSwitchSdk` instance, just change the values of the particular field in the collection. With that new object, call the send feedback method:
+
+```swift
+sdk.sendFeedback(feedback)
+```
+
+After that, the SDK will send the updated fields to our backend. Upon completion the
+
+```swift
+func switchSdkDidSendFeedback(sdk:GiniSwitchSdk)
+```
+
+will be invoked. Please note that it will be called only if the feedback is successfully received by the backend. if not, the
+
+```swift
+func switchSdk(sdk:GiniSwitchSdk, didReceiveError error:Error)
+```
+
+will be called with an error type of `feedbackError`.
+
+Another caveat to sending feedback is that you will have to keep the SDK "alive" until the request goes through. As already discussed in the "Going back" section, even after the SDK's UI is dismissed and the `switchSdkDidComplete` delegate method is invoked, the SDK still cannot be terminated and left to ARC to be cleared.
+
+To properly dispose of the SDK after feedback is sent, wait until the `switchSdkDidSendFeedback` or the `didReceiveError` (with a `feedbackError` error type) to be invoked. Once that happens, terminate the SDK as described in "Dismiss the SDK".
```

Comment:
btw. you can create anchors `# <a name="dismiss-sdk"/>Dismiss the SDK` in github markdown and link to them `[Dismiss the SDK](#dismiss-sdk)`